RestApi/Controllers/ExpenseCategoryController.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using RestApi.Domain;
+
+[ApiController]
+[Route("api/[controller]")]
+public class ExpenseCategoryController : ControllerBase
+{
+    private static List<ExpenseCategory> _defaultCategories = new List<ExpenseCategory>
+    {
+        new ExpenseCategory { Name = "Electronics" },
+        new ExpenseCategory { Name = "Transport" },
+        new ExpenseCategory { Name = "Products" }
```

Comment:
What about taking the default category values from the database during startup? This way, we can dynamically load those defaults while booting the app.